test/Approximations/overapproximate_expmap.jl

```diff
@@ -61,3 +78,17 @@ for N in @tN([Float32, Float64])
         end
     end
 end
+
+let 
+    # This test for MatrixZonotopeExp is only ran with Float64 since it is expensive
+    # and requires very high taylor order `k` to pass on Float32
+    N=Float64
```

Comment:
```suggestion
for N in @tN([Float64])
    # This test is only run with Float64 since it is expensive
    # and requires high Taylor order `k` to pass on Float32
```